internal/pkg/plugin/gitlabci/java/template.go

```diff
@@ -0,0 +1,72 @@
+package java
+
+import (
+	"bytes"
+	"html/template"
+)
+
+var gitlab_ci_template = `
+image: docker:stable
```

Comment:
1. Use `go:embed` to highlight the template
2. go's constant naming style is `CamelCase`

Reply:
I'll modify `gitlab_ci_template` to `gitlabCITemplate`. What do you think? 

`gitlabCITemplate` is a string here, I don not think we should use `go:embed`.

Reply:
But I don't know why not use` embed`😂? `yaml` is not rendered with highlighting in ``, and it is obviously easier to edit by extracting it to a separate file.

Reply:
Agree +1, I'll refactor this.